Zend/zend_builtin_functions.stub.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+function zend_version(): string {}
+
+function func_num_args(): int {}
+
+/** @return mixed */
+function func_get_arg(int $arg_num) {}
+
+/** @return array|false */
+function func_get_args() {}
+
+function strlen(string $str): int {}
+
+function strcmp(string $str1, string $str2): int {}
+
+/** @return int|false */
+function strncmp(string $str1, string $str2, int $len) {}
+
+function error_reporting($new_error_level = UNKNOWN): int {}
+
+function define(string $constant_name, $value, bool $case_insensitive = false): bool {}
```

Comment:
This doesn't match https://www.php.net/manual/en/function.define.php#refsect1-function.define-changelog ?

> case_insensitive has been deprecated and will be removed in version 8.0.0.

Reply:
The argument is still there, but setting it to true will always generate an error.